crates/shielder-scheduler-server/README.md

```diff
@@ -35,18 +35,16 @@ The server maintains proper security by verifying the KMS key relationship with
 
 ### Local Development
 
-For local development and testing, you can use the `--disable-kms` command line flag to skip AWS KMS integration:
+For local development and testing, you can use the cargo feature `local-run` to skip AWS KMS integration and use in-memory storage instead of DynamoDB:
```

Comment:
This README lacks section about DynamoDB and assumptions around it (e.g. table name must be unique withing region etc)